src/content/ui-api/rest-api/index.md

```diff
@@ -18,12 +18,10 @@ last_review_date: 2021-01-01
 
 # The Giant Swarm REST API
 
-The Giant Swarm REST API is an abstraction over the many resources that reside in the management cluster of a Giant Swarm installation. It is used by our dedicated user interfaces, the web user interface and gsctl.
+The REST API has been designed originally to provide a simple, easy to use interface to the relevant resources for managing clusters, key pairs, etc. while keeping the internals under the hood.
 
-Browse our [API documentation](/api/) for a complete overview into the provided functionality.
+Since its inception, at Giant Swarm we learned that there are always more use cases emerging on our customer's side than we could anticipate in our REST API design. We realized that the best we can do for you to provide full insight into the state and spec of your infrastructure is by opening up the underlying system itself.
```

Comment:
```suggestion
Since its inception at Giant Swarm, we learned that there are always more use cases emerging on our customer's side than we could anticipate in our REST API design. We realized that the best we can do to provide full insight into the state and spec of your infrastructure is by opening up the underlying system itself.
```